packages/commonwealth/client/scripts/controllers/app/recent_activity.ts

```diff
@@ -68,95 +64,6 @@ class RecentActivityController {
       return { info, count };
     });
   }
-
-  public getMostActiveUsers() {
-    return this._activeUsers;
-  }
-
-  public async getRecentTopicActivity(id?: string): Promise<Thread[]> {
```

Comment:
this was not used anywhere -- stale code

Reply:
You are correct it was used in IChainAdapter, but the calculation was unnecessary because the output (overview/pinned threads) was not correctly used anywhere (properly) -- in the migration we updated it.